artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -355,10 +355,27 @@ default Message setValidatedUserID(String validatedUserID) {
 
    String getAddress();
 
+   /**
+    * Notice the address in AMQP or other protocols
+    * is only effective if the message is copied or reencoded.
```

Comment:
Is this true now for AMQP messages? Do the 'extra properties' set for the AMQP message actually influence the 'to' address anymore?

Reply:
no.. not any more...


will remove this.